database/migrations/2021_07_27_062215_create_category_material_table.php

```diff
@@ -13,7 +13,7 @@ class CreateMaterialCategoryTable extends Migration
      */
     public function up()
     {
-        Schema::create('material_category', function (Blueprint $table) {
+        Schema::create('category_material', function (Blueprint $table) {
```

Comment:
in laravel convention, you should write the tables names according to their order in migrations , so  the material comes before category